src/appmain.js

```diff
@@ -113,133 +114,144 @@ class App extends LitElement
     }
     return html`
     ${!this.embed || this.embed === "full" ? html`
-      <nav class="navbar has-background-info" role="navigation" aria-label="main navigation">
+    <a href="#skip-main-target" @click=${this.skipMenu} class="skip-link">Skip main navigation</a>
+    <nav class="navbar has-background-info" aria-label="main">
       <div class="navbar-brand">
-        ${!this.embed ? html`
-        <a class="navbar-item wr-logo-item" title="ReplayWeb.page" href="/">
-          <fa-icon id="wrlogo" size="2.5rem" .svg=${rwpLogo}></fa-icon>
-        </a>` : html`
-        <a class="navbar-item wr-logo-item" title="ReplayWeb.page">
-          <fa-icon id="wrlogo" size="2.5rem" .svg=${rwpLogo}></fa-icon>
-        </a>
+        ${!this.embed ? html `
+          <a href="/" class="navbar-item wr-logo-item menu-only" aria-labelledby="home">
+            <fa-icon id="wrlogo" size="2.5rem" .svg=${rwpLogo} aria-hidden="true"></fa-icon>
+          </a>
+          <span class="navbar-item wr-logo-item wide-only">
+            <fa-icon id="wrlogo" size="2.5rem" .svg=${rwpLogo} aria-hidden="true"></fa-icon>
+          </span>`: html `
+          <span class="navbar-item wr-logo-item">
+            <fa-icon id="wrlogo" size="2.5rem" .svg=${rwpLogo} aria-hidden="true"></fa-icon>
+          </span>
         `}
-
-        <a role="button" @click="${this.onNavMenu}"
-        class="navbar-burger burger ${this.navMenuShown ? 'is-active' : ''}" aria-label="menu" aria-expanded="false">
+        <a href="#" role="button" id="menu-button" @click="${this.onNavMenu}" @keyup="${this.clickOnSpacebarPress}"
```

Comment:
I should extract clickOnSpacebarPress to a pageutil, instead of duplicating everywhere.